BibBuddy/app/src/main/java/de/bibbuddy/ApiReader.java

```diff
@@ -0,0 +1,45 @@
+package de.bibbuddy;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.util.stream.Collectors;
+
+public class ApiReader implements Runnable {
+
+    private String metadata = null;
+    private String url;
+
+    /**
+     * The ApiReader class reads and returns data from the API
+     *
+     * @author Luis Moßburger
+     *
+     * @param url        concrete url that should be sent to API
+     */
+    ApiReader(String url) {
+        this.url = url;
+    }
+
+    public void run() {
+        URL metadataUrl;
+        BufferedReader in;
+        String metadata = "";
+        System.out.println(url);
+        try {
```

Comment:
Please add a newline for better readability.
```suggestion

        try {
```

Reply:
Thanks! Adressed in latest commit.